src/stanc/stanc.ml

```diff
@@ -83,6 +84,9 @@ let options =
     ; ( "--auto-format"
       , Arg.Set pretty_print_program
       , " Pretty prints the program to the console" )
+    ; ( "--canonize"
```

Comment:
Hahaha. What about `--print-canonical` or something like that?

Reply:
sure, sounds good.